joomla/joomla-cms
# [4.0] Update node dependencies

Large npm dependency update

### Summary of Changes
Updates all our dependencies to latest and greatest versions

### Testing Instructions
Generally navigate around and check for new javascript/css errors as all the minifers/compilers have recieved updates.

Check TinyMCE
Check Codemirror
Check Backend Menu
Check Media Manager

@dgrammatiko I'd appreciate if you could check the change I had to make to the build tools. It came off the back of the issue mentioned here https://github.com/babel/minify/issues/904 but I have to admit I'm not clear what the implications exactly are

### Documentation Changes Required
None
